services/communitytokens/estimations.go

```diff
@@ -356,11 +356,13 @@ func (s *Service) mintTokensGasUnits(ctx context.Context, chainID uint64, contra
 }
 
 func (s *Service) prepareCommunityTokenFees(ctx context.Context, from common.Address, to *common.Address, gasUnits uint64, chainID uint64) (*CommunityTokenFees, error) {
-	suggestedFees, err := s.feeManager.SuggestedFeesGwei(ctx, chainID)
+	suggestedFeesWei, err := s.feeManager.SuggestedFees(ctx, chainID)
 	if err != nil {
 		return nil, err
 	}
 
+	suggestedFees := router.ConvertFeesToGwei(suggestedFeesWei)
```

Comment:
Not sure why would we want to expose such function? exposing an API with gwei and wei is fine no?